### PR TITLE
Add Galaxy Webhook slides

### PR DESCRIPTION
**Pull request by blankclemens**

### CODE REVIEW

**Comment by martenson** [comment]
Looks good. Maybe add a link to the docs? https://docs.galaxyproject.org/en/latest/admin/special_topics/webhooks.html

File: Dev-Corner/slides/webhooks.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+topic_name: "Dev-Corner"
+tutorial_name: webhooks
+logo: "GTN"
+---
+
+### What are Galaxy Webhooks?
+
+A system which can be used to write small JS and/or Python functions to change predefined locations in the Galaxy client
+<br><br>
+In short: A plugin infrastructure for Galaxy
```
Comment (bgruening) [inline Dev-Corner/slides/webhooks.html]: for the Galaxy User Interface.

File: metadata/Dev-Corner.yml [hunk 2]
```diff
@@ -116,11 +116,35 @@ material:
...
+
+    objectives:
+    - "Discover what Galaxy Webhooks are"
+    - "Be able to create Webhooks"
+    - "Be able to add a Galaxy Webhook in a Galaxy instance"
+
+    time_estimation: "1h"
+
+    key_points:
+    - "A Galaxy Webhook provides an easy way to customize Galaxy with plugins"
```
Comment (bgruening) [inline metadata/Dev-Corner.yml]: Galaxy UI

**Comment by blankclemens** [comment]
Thanks, good points!

**Comment by bgruening** [comment]
Cool, thanks!

**Pull request merged**